src/app/content/genericstyles.screenshotspec.ts

```diff
@@ -0,0 +1,40 @@
+/** @jest-environment puppeteer */
+import {
+  fullPageScreenshot,
+  navigate,
+  setDesktopViewport,
+  setMobileViewport
+} from '../../test/browserutils';
+
+const CHEMISTRY_BOOK = '/books/chemistry-2e';
```

Comment:
Rather than relying on the chemistry book being built for the unit tests, tests would run faster and any changes would be clearer if a "fake" book was used for screenshot tests.

[src/app/components/NavBar/index.screenshotspec.ts#L9](https://github.com/openstax/rex-web/blob/master/src/app/components/NavBar/index.screenshotspec.ts#L9) can serve as an example.